silex_client/network/websocket_dcc.py

```diff
@@ -0,0 +1,17 @@
+import socketio
+
+from silex_client.utils.log import logger
+
+
+class WebsocketDCCNamespace(socketio.AsyncClientNamespace):
+    def __init__(self, namespace: str, context_metadata: dict, url:str):
+        super().__init__(namespace)
+        self.context_metadata = context_metadata
+        self.url = url
+
+    async def on_connect(self):
+        logger.info("Connected to %s", self.url)
+        await self.emit("initialisation", self.context_metadata)
+
+    async def on_disconnect(self):
```

Comment:
on disconnect also emit a message?

Reply:
The server can use the on disconnect event too